locksmith/src/routes/v2/ticket.ts

```diff
@@ -71,4 +83,6 @@ router.get(
 
 router.get('/:network/lock/:lockAddress/key/:keyId', getTicket)
 
+router.get('/:network/lock/:lockAddress/:eventSlug/key/:keyId', getTicket)
```

Comment:
Please change the slug to be consistent with the `put` call above and consistent with REST principles.

```suggestion
router.get('/:eventSlug/:network/lock/:lockAddress/key/:keyId/check', getTicket)
```